agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -139,6 +191,8 @@ for item in ${sysinfo//,/ };do
 		collect_block &
 	elif [[ "$item" == "ara" ]]; then
 		collect_ara_data &> $pbench_log
+	elif [[ "$item" == "stockpile" ]]; then
+		collect_stockpile_data
```

Comment:
Shouldn't this invocation be put in the background like the other sysinfo options?